module/iam/outputs.tf

```diff
@@ -0,0 +1,3 @@
+output "arn" {
+    value = "${aws_iam_role.this.arn}"
+}
```

Comment:
no hay necesidad de agregar `"${}"` pueden usar `aws_iam_role.this.arn` directamente